public/components/common/helpers/ppl_docs/language_structure/datatypes.ts

```diff
@@ -5,226 +5,151 @@
-[Timestamp](#timestamp).
-
-| Type     | Syntax                             | Range                                                        |
-|----------|------------------------------------|--------------------------------------------------------------|
-| Datetime | 'yyyy-MM-dd hh:mm&#58;ss\[.fraction\]' | '0001-01-01 00:00:00.000000' to '9999-12-31 23:59:59.999999' |
-
-#### **Timestamp**
-
-A timestamp instance is an absolute instant independent of timezone or
-convention. For example, for a given point of time, if we set the
-timestamp of this time point into another timezone, the value should
-also be different accordingly. Besides, the storage of timestamp type is
-also different from the other types. The timestamp is converted from the
-current timezone to UTC for storage, and is converted back to the set
-timezone from UTC when retrieving.
-
-| Type      | Syntax                             | Range                                                            |
-|-----------|------------------------------------|------------------------------------------------------------------|
+| Type     | Syntax       | Range                        |
+|----------|--------------|------------------------------|
+| \`date\` | 'yyyy-MM-dd' | '0001-01-01' to '9999-12-31' |
+
+#### Time
+
+The \`time\` data type represents the time of day as displayed on a clock or watch, without specifying a particular time zone. It does not include any information about the calendar date.
+
+| Type     | Syntax       | Range                        |
+|----------|--------------|------------------------------|
+| \`time\` | 'hh:mm&#58;ss\[.fraction\]' | '00:00:00.000000' to '23:59:59.999999' |
+
+#### Datetime
+
+The \`datetime\` data type represents a combination of \`date\` and \`time\`. The \`datetime\` data type does not contain time zone information. For an absolute time point that contains both datetime and time zone information, see the [Timestamp](#timestamp) section. 
+
+See the [Conversion between date and time types](#conversion-between-date-and-time-types) section for information about the conversion rule for \`date\` or \`time\` to \`datetime\`.
+
+| Type     | Syntax | Range |
+|----------|--------|-------|
+| \`datetime\` | 'yyyy-MM-dd hh:mm:ss\[.fraction\]' | '0001-01-01 00:00:00.000000' to '9999-12-31 23:59:59.999999' |
+
+#### Timestamp
+
+The \`timestamp\` data type represents absolute points in time, unaffected by time zones or conventions. The \`timestamp\` data type differs from other data types in its storage and retrieval behavior. When a timestamp is sorted, it is converted from Coordinated Universal Time (UTC) to the specified time zone. Conversely, when a timestamp is retrieved, it is converted back to UTC before being displayed or used in calculations. This ensures the timestamp values remain consistent and comparable across different time zones.
+
+| Type      | Syntax | Range |
+|-----------|--------|-------|
```

Comment:
@pjfitzgibbons Several examples use `&#58;01`. What should these examples actually look like?